File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        char[] charArr = my_string.toCharArray();
        
        for(char c:charArr)
            if(Character.isDigit(c))
                answer += Integer.parseInt(String.valueOf(c));
        
        
        return answer;
    }
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    var answer = 0;
    var _my_string = my_string.split("");
    
    for(var s of _my_string){
        var num = parseInt(s);
        if(isNaN(num)){
            continue;
        }
        answer += parseInt(num);
    }
    
    return answer;
}